integration/app/app.config.ts

```diff
@@ -42,7 +41,7 @@ export const appConfig: ApplicationConfig = {
       [TodosState, TodoState],
       { developmentMode: !environment.production, selectorOptions: {} },
       withNgxsFormPlugin(),
-      withNgxsLoggerPlugin({ logger: console, collapsed: false, disabled: true }),
+      withNgxsLoggerPlugin({ logger: console, collapsed: false, disabled: false }),
```

Comment:
Do we want to enable logging as part of this PR?
Or was this just left after some debugging?